src/AddToMetaMask.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+const AddToMetaMask = (props: {
+  name: string,
+  chainId: string,
+  token: string,
+  rpcs: string[],
+  be: string[],
+}) => {
+  return (
+    <span>
+      {props.rpcs.map(item => (
+        <code>{item}<br/></code>
```

Comment:
I expect this is showing console errors during development. It should
Needs `key` attribute